src/components/network-chart/index.css

```diff
@@ -4,10 +4,73 @@
 
 .networkChart {
   display: flex;
-  flex: 1;
   flex-flow: column nowrap;
+  flex: 1;
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  user-select: none;
+  cursor: default;
 }
 
-.networkChartCanvas {
+.networkChart .treeViewBody .treeViewBodyInnerWrapper {
+  flex-flow: column nowrap;
+  width: 100%;
   border-top: 1px solid var(--grey-30);
 }
+
+
+.networkChartRowItem {
+  width: 100%;
+  height: 1.5em;
```

Comment:
The height of the items in the list is defined in pixels:

```
const ROW_HEIGHT = 17.5;
```

But here it is defined in `em`s. I think it would be safer to define all of the sizing here as pixels, since the `em` units are relative to their surroundings. This seems like something that could break unexpectedly due to code changes, or the relative sizing due to different CSS configurations.

Normally, I prefer `em`s in most of my webpages, but here we've been mostly using `px` for sizing. It might be better to consolidate these sizings to `px` since that is how we are defining the sizing in the JS code. I'll point out the other `em`s I see as well.